resource_templates/cluster_group_scoped_kustomization.tf

```diff
@@ -0,0 +1,34 @@
+// Create/ Delete/ Update Tanzu Mission Control cluster group scoped kustomization entry
+resource "tanzu-mission-control_kustomization" "test_kustomization" {
+  name = "test"
+
+  namespace_name = "test"
+
+  org_id = "test" // optional
+
+  scope {
+    cluster_group {
+      name = "test"
+    }
+  }
+
+  spec {
+    source {
+      namespace = "test" // Namespace of the repository. (can be referenced in Tf - to explicitly call delete on this resource if the dependency is deleted)
+
+      name = "test" // Name of the repository. (can be referenced in Tf - to explicitly call delete on this resource if the dependency is deleted)
+    }
+
+    path = "/"
+
+    prune = true
+
+    interval = "5m" // can be "5s" etc.
+
+    target_namespace = "test"
+  }
+
+  status = {
```

Comment:
status cannot be set in the resource .. this should only be computed

Reply:
Agreed, updated the same.